packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -121,6 +162,46 @@ void main() {
     expect(box.size.height - oldHeight, greaterThanOrEqualTo(100.0)); // 100 + some margin
   });
 
+  testWidgets('ExpansionPanelList does not merge header when canTapOnHeader is false', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: ExpansionPanelListSemanticsTest(),
+      ),
+    );
+    final SemanticsHandle handle = tester.ensureSemantics();
```

Comment:
Do this as the first thing in the test. That way you can avoid the extra pump in the next line.